Client/Frontend/Settings/Brave Today/BraveTodayAddSourceResultsViewController.swift

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
```

Comment:
nit: 2021

Reply:
Ah that's because our custom header is hard-coded, I'll open up a separate PR to update it, but all files created since Jan will have 2020